builds/e2e/templates/nested-longhaul-deploy.yaml

```diff
@@ -0,0 +1,166 @@
+        
+        echo "##vso[task.setvariable variable=TrustBundle]/certs/certs/azure-iot-test-only.root.ca.cert.pem"
+        echo "##vso[task.setvariable variable=DeviceCaCert]/certs/certs/iot-edge-device-${deviceId}-full-chain.cert.pem"
+        echo "##vso[task.setvariable variable=DeviceCaKey]/certs/private/iot-edge-device-${deviceId}.key.pem"
+  - task: Bash@3
+    displayName: 'Run Long Haul Deployment'
+    inputs:
+      targetType: inline
+      script: |
+        declare -a cnreg=( ${{ parameters['container.registry.credential'] }} )
+        . $(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}/artifactInfo.txt
+        testName="LongHaul"
+        
+        testInfo="TestBuildNumber=${{ parameters['test.buildNumber'] }}"
+        testInfo="$testInfo,TestBuildId=${{ parameters['test.buildId'] }}"
+        testInfo="$testInfo,BuildSourceBranch=${{ parameters['build.source.branch'] }}"
+        testInfo="$testInfo,EdgeletSourceBranch=${{ parameters['edgelet.source.branch'] }}"
+        testInfo="$testInfo,ImagesSourceBranch=${{ parameters['images.source.branch'] }}"
+        testInfo="$testInfo,HostPlatform=${{ parameters['metricsCollector.hostPlatform'] }}"
+        testInfo="$testInfo,TestName=$testName"
+        
+        chmod +x $(Build.SourcesDirectory)/scripts/linux/trcE2ETest.sh
+        
+        sudo --preserve-env $(Build.SourcesDirectory)/scripts/linux/trcE2ETest.sh \
+            -testDir "$(Agent.HomeDirectory)/.." \
+            -releaseLabel "${{ parameters['release.label'] }}" \
+            -artifactImageBuildNumber "$BuildNumber" \
+            -testName "$testName" \
+            -testMode "$testName" \
+            -containerRegistry "${{ parameters['container.registry'] }}" \
+            -containerRegistryUsername "${{ parameters['container.registry.username'] }}" \
+            -containerRegistryPassword "${{ parameters['container.registry.password'] }}" \
+            -iotHubConnectionString "${{ parameters['iotHub.connectionString'] }}" \
+            -eventHubConnectionString "${{ parameters['eventHub.connectionString'] }}" \
+            -testDuration  24:00:00 \
+            -testStartDelay "${{ parameters['test.startDelay'] }}" \
+            -loadGenMessageFrequency "${{ parameters['loadGen.message.frequency'] }}" \
+            -logAnalyticsWorkspaceId "${{ parameters['logAnalytics.workspaceId'] }}" \
+            -logAnalyticsSharedKey "${{ parameters['logAnalytics.sharedKey'] }}" \
+            -logAnalyticsLogType "${{ parameters['logAnalytics.logType'] }}" \
+            -metricsEndpointsCSV "${{ parameters['metricsCollector.metricsEndpointsCSV'] }}" \
+            -metricsScrapeFrequencyInSecs "${{ parameters['metricsCollector.scrapeFrequencyInSecs'] }}" \
+            -metricsUploadTarget "${{ parameters['metricsCollector.uploadTarget'] }}" \
+            -deploymentFileName "long_haul_deployment.template.json" \
```

Comment:
Are we still using the same deployment file for the LongHaul nested edge without any modification? API Proxy? Bridge?

Reply:
Not sure. @huguesBouvier any idea?

Reply:
@richma-ms I don't recall what we have discussed regarding what tests are in Nested Edge Long Haul.

Reply:
Will create another task to track this